emission/net/api/cfc_webapp.py

```diff
@@ -59,6 +59,7 @@
     config_file = open('conf/net/api/webserver.conf.sample')
 
 OPENPATH_URL="https://www.nrel.gov/transportation/openpath.html"
+STUDY_CONFIG = os.getenv('STUDY_CONFIG', "stage-program")
```

Comment:
You should clarify how the `STUDY_CONFIG` is set and add a sample.

Reply:
You should document what happens if the study config is not set, so that we can check for it when we deploy and fix ASAP if it is broken.